### DISCUSSION TITLE
Alternative Installation with apache

### DISCUSSION THREAD

**Discussion by flashlab** [VueTorrent/VueTorrent#1595] [Show and tell]
With mod proxy and rewrite of apache enabled, we can deploy VueTorrent separately.
Just create `.htaccess` in the `public` folder:
```
RewriteEngine On
RewriteCond %{REQUEST_FILENAME} !-f
RewriteRule ^(api.*)$ http://localhost:WEBUI_PORT/$1 [P,L]
```

**Comment by Larsluph** [comment]
Added to [wiki alt installation methods](https://github.com/VueTorrent/VueTorrent/wiki/Alternative-Installation-Methods)
